proxy/src/main/java/com/wavefront/agent/AbstractAgent.java

```diff
@@ -1153,6 +1160,14 @@ private void postProcessConfig() {
       }
     }
 
+    // Create set of trace derived RED metrics custom Tag keys.
+    if (!StringUtils.isBlank(traceDerivedCustomTagKeysProperty)) {
+      String[] derivedMetricTagKeys = traceDerivedCustomTagKeysProperty.split(",");
+      for (String tag : derivedMetricTagKeys) {
+        traceDerivedCustomTagKeys.add(tag.trim());
+      }
+    }
+
```

Comment:
```
if (traceDerivedCustomTagKeysProperty != null) {
  traceDerivedCustomTagKeys.addAll(Splitter.on(",").omitEmptyStrings().trimResults().split(traceDerivedCustomTagKeysProperty));
}
```
pattern we tend to use is less verbose, but that's a matter of personal taste mostly, so no big deal.

Reply:
We decided to keep this unchanged.